### ISSUE TITLE
multi-note editor not available

### ISSUE THREAD

**Issue by aisbergde** [notable/notable#1168]
in the tutorial is a note 
09 - Multi-Note Editing
but this feature is not available in my version, what am I doing wrong?
Windows, version 1.8.4

When I multi select 2 or more notes I get only:
"2 notes selected"
and below some buttons or fields to add tags, delete, mark es favorite and so on
but there is no way to edit multiple notes

Windows, version 1.8.4

**Comment by fabiospampinato** [comment]
> When I multi select 2 or more notes I get only:
"2 notes selected"
> and below some buttons or fields to add tags, delete, mark es favorite and so on
> but there is no way to edit multiple notes

That's the multi-note editor really, it's intended more to edit the metadata of multiple notes at once but not their content really.

What kind of multi-editing did you want to do? Something like a search and replace maybe? If so this isn't implemented yet within the app, but as your notes as just files on disk you can open your notes directory with another tool, like vs code, and perform that action there.

**Comment by aisbergde** [comment]
I misunderstand the word "editor": I thought there are are 2 windows with code and I can compare code for example. But I don't really need this, VS Code also works great as markdown editor and there I can open several windows if I need.

may be "multi-note metadata editor" would be a better to understand name for this feature. 

What I miss a little bit is a short look on the hidden area with metadata on top of the file. Need to open files in separate application...

But in General I am very impressed about notable: 
I was searching a markdown based notes taking system based on simple markdown files, which I can also synchronize with Android. And Notable is what I need. It has even the same editor and preview like VSC, what a great idea!

Sorry for off topic:
Now I will investigate how to connect notable with my todo.txt based todo system. 
I see, you also have a todo VSC plugin, but for me it is not (yet) clear how this could be used on Windows and on Android. 
If you are the developer of notable and todo, maybe there is already a solution?

**Comment by fabiospampinato** [comment]
> I misunderstand the word "editor": I thought there are are 2 windows with code and I can compare code for example. 

Yeah that's a bit confusing put it like that 🤔 Maybe I should rename the thing.

In v1.9 there'll be support for multiple tabs: https://www.reddit.com/r/Notable/comments/gs3ul6/preview_redesign_multitag_selection_tabs_quick/ but still only one tab will be active at any give time, this will eventually be expanded and the thing will just work like in vs code.

> What I miss a little bit is a short look on the hidden area with metadata on top of the file.

We are keeping track of adding a separate editor for the metadata here: #67, I've made quite a few changes recently in the app, this should be easily implementable now. Why do you need to view the raw metadata? Like are there some pieces of that metadata that you'd like the app to display better?

> I see, you also have a todo VSC plugin, but for me it is not (yet) clear how this could be used on Windows and on Android.
> If you are the developer of notable and todo, maybe there is already a solution?

I definitely need to implement more task-management related features in Notable, right now you can basically just write a toggleable task in this like this:

```
- [ ] Task
```

There are definitely some good ideas that we can borrow from my todo extension, I haven't worked on that yet.

**Comment by aisbergde** [comment]
> We are keeping track of adding a separate editor for the metadata here: #67, I've made quite a few changes recently in the app, this should be easily implementable now. Why do you need to view the raw metadata? Like are there some pieces of that metadata that you'd like the app to display better?

- I am a control freak
- I would like to see all assigned tags, normally I would use this syntax: `+abc +def/ghi` inside the text
- I am a lazy man:  
I would like to copy and paste tags from one file into another because when I enter new tags I have to enter each one by one and there is even no list to select from existing tag
- some automatic tag detection even would be better:  
for example if the text contains `+abc +def/ghi` I would like Notable to scan my the content and to automatic assign tags "abc" and "def/ghi"

if you like my ideas I can create separate feature requests in separate issues :-)

**Comment by fabiospampinato** [comment]
> I would like to see all assigned tags, normally I would use this syntax: +abc +def/ghi inside the text

Let's track this here: #105 

> I would like to copy and paste tags from one file into another because when I enter new tags I have to enter each one by one and there is even no list to select from existing tag

That'd be useful 🤔 Ideally there should be tags autocompletion (#20) but even then duplicating a lot of tags wouldn't be particularly quick. I could add a specific command for this too 🤔 

> for example if the text contains +abc +def/ghi I would like Notable to scan my the content and to automatic assign tags "abc" and "def/ghi"

That sounds like something a specific plugin could be written for, in the future.

> if you like my ideas I can create separate feature requests in separate issues :-)

That'd be useful, thanks. Maybe you can open issues for in-note tags and a command for copying all tags, the rest I think it's tracked elsewhere already.
